client/hwlib/src/collectors/cpuinfo.rs

```diff
@@ -2,16 +2,16 @@
  * All rights reserved.
  *
  * This program is free software: you can redistribute it and/or modify
- * it under the terms of the GNU General Public License as published by
+ * it under the terms of the GNU Lesser General Public License as published by
```

Comment:
Looking at https://sites.google.com/a/canonical.com/operations/legal/software-licensing it seems that we should also include the license version with it?